src/pipelines/docs.ml

```diff
@@ -238,7 +240,7 @@ let v ~config ~api ~opam () =
                |> Current.state ~hidden:true)
         |> Current.list_seq
         |> Current.map (List.filter_map Result.to_option)
-      and+ metadata = metadata in
+      and+ metadata = pages in
```

Comment:
Is `metadata` replaced by `pages` ? In that case I believe the unused variable / module should be removed

Reply:
Ah yes, metadata should be used by pages, I need to fix that.